### PR TITLE
Add GraphQL Pokemon API to Games & Comics

### PR DESCRIPTION
**Pull request by favna**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not have more than 100 characters
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -809,6 +809,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Geek-Jokes](https://github.com/sameerkumar18/geek-joke-api) | Fetch a random geeky/programming related joke for use in all sorts of applications | No | Yes | Yes |
 | [Genshin Impact](https://genshin.dev) | Genshin Impact game data | No | Yes | Yes |
 | [Giant Bomb](https://www.giantbomb.com/api/documentation) | Video Games | `apiKey` | Yes | Unknown |
+| [GraphQL Pokemon](https://graphqlpokemon.favware.tech) | GraphQL powered Pokemon API. Supports generations 1 through 8 | No | Yes | Yes |
```
Comment (matheusfelipeog) [inline README.md:812]: I believe the link to the official repository is better: https://github.com/favware/graphql-pokemon

It contains more information about the project.

```suggestion
| [GraphQL Pokemon](https://github.com/favware/graphql-pokemon) | GraphQL powered Pokemon API. Supports generations 1 through 8 | No | Yes | Yes |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @favna <3

Can you resolve this review, please?

File: README.md (hunk 1 above)
Comment (favna) [reply README.md:812]: Ah yeah, that's fair. I wasn't sure which of the 2 to put there.

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok @favna. Thanks for the update <3

**Pull request merged**